docs/CONTRIBUTORS.rst

```diff
@@ -125,6 +125,26 @@ increase log level up to DEBUG ('-vvvv').
     $ python test_updater.py -vvvv TestUpdater.test_4_refresh # run test with DEBUG log level
 
 
+All of the log levels and the corresponding options that could be used for testing are:
+
+.. list-table::
+   :widths: 20 25
+   :header-rows: 1
+
+   * - Option
+     - Log Level
+   * - default (no argument passed)
+     - ERROR
+   * - `-v`
+     - ERROR + unittest module prints test names
```

Comment:
are test names only printed at this log level?

Reply:
Yes.
Example: running `python test_updater.py` from the tests directory on the development branch leads to the output: 
![image](https://user-images.githubusercontent.com/16246778/94701069-e33cbc80-0344-11eb-9d62-6ce94c420dc2.png)


but running `python test_updater.py -v` from the tests directory on the development branch adds the test names
![image](https://user-images.githubusercontent.com/16246778/94700886-abce1000-0344-11eb-9f64-8d8dab2ddede.png)